vcap/vcap/device_mapping.py

```diff
@@ -83,15 +84,49 @@ def filter_func(devices):
         return DeviceMapper(filter_func=filter_func)
 
     @staticmethod
-    def map_to_all_myriad(cpu_fallback=True):
-        """Returns Myriad device. If cpu_fallback=True, it will return CPU:0
-        if there are no myriad devices."""
+    def map_to_openvino_devices():
+        """Intelligently load capsules onto available OpenVINO compatible
+        devices.
+
+        Since support for OpenVINO devices is very experimental, there is a
+        temporary environment variable being added to whitelist devices
+        specifically. This variable will be deprecated and removed after a
+        short testing period.
+
+        OPENVINO_ALLOWABLE_DEVICES can be
+            "", "HDDL", or "MYRIAD"
+        The device "CPU" is _always_ allowed and always loaded onto.
+
+        The logic is as follows: IECore().available_devices is used to scan
+        for devices, then the following logic is used to load onto devices,
+        as long as the devices are whitelisted by OPENVINO_ALLOWABLE_DEVICES
+
+        Here are the cases:
+        ['CPU:0', 'MYRIAD', ...] => ['CPU:0', 'MYRIAD']
+            If MYRIAD is whitelisted. Else, it will return ["CPU:0"]
+        ['CPU:0', 'HDDL', ...] =>  ['CPU:0', 'HDDL']
+            If HDDL is whitelisted. Else, it will return ["CPU:0"]
+        ['CPU:0'] => ['CPU:0']
+            Always load onto CPU.
+        """
 
         def filter_func(devices):
-            myriad_devices = [d for d in devices
-                              if d.lower().startswith("myriad")]
-            if not myriad_devices and cpu_fallback:
-                return ["CPU:0"]
-            return myriad_devices
+            allowed_device_type = os.environ.get(
+                "OPENVINO_ALLOWABLE_DEVICES", None)
+
+            allowed_devices = []
+            if allowed_device_type is None or allowed_device_type == "":
```

Comment:
`if allowed_device_type in [None, ""]` might be cleaner

I prefer Lists in this use case, but stay consistent with the `in` below that uses a Tuple.